File: solutions/c/raindrops/1/raindrops.c
#include "raindrops.h"

void convert(char result[], int drops) {
	if (!(drops % 3)) strcat(result, "Pling");
	if (!(drops % 5)) strcat(result, "Plang");
	if (!(drops % 7)) strcat(result, "Plong");
	if (!*result) sprintf(result, "%d", drops);
}

File: solutions/lua/raindrops/1/raindrops.lua
return function(n)
	local message = ""
	if n % 3 == 0 then
		message = message .. "Pling"
	end
	if n % 5 == 0 then
		message = message .. "Plang"
	end
	if n % 7 == 0 then
		message = message .. "Plong"
	end
	if message == "" then
		return tostring(n)
	else
		return message
	end
end
